pkg/apis/core/types.go

```diff
@@ -1232,6 +1232,9 @@ type DownwardAPIProjection struct {
 type AzureFileVolumeSource struct {
 	// the name of secret that contains Azure Storage Account Name and Key
 	SecretName string
+	// the namespace of the secret that contains Azure Storage Account Name and Key
+	// +optional
+	SecretNamespace *string
```

Comment:
that requires api change, since in-tree driver is already in migration state, it's not recommended to change this. For in-tree driver, secretNamespace should be `default`.